src/handler/analytics.js

```diff
@@ -1,4 +1,5 @@
 const GA_ENDPOINT = "https://www.google-analytics.com/mp/collect";
+const KEY_URL = "https://europe-west1-room-elephnat-key-retriever.cloudfunctions.net/getKeys";
```

Comment:
Please check the service for a typo in elephant.

Reply:
done.